### ISSUE TITLE
Package request: Navidrome

### ISSUE THREAD

**Issue by aciceri** [NixOS/nixpkgs#88903]
**Project description**
A music server and streamer compatible with Subsonic/Airsonic.

**Metadata**

* homepage URL: https://www.navidrome.org/
* source URL: https://github.com/deluan/navidrome/
* license: gpl3

**Comment by andrea96** [comment]
I created a fork of `nixpkgs` with a branch called `navidrome` that has this application: https://github.com/andrea96/nixpkgs/tree/navidrome

I also added myself to the mantainers.

Now I have two questions:
- How can I correctly create a pull request? 
- My derivation doesn't compile the project but download the statically compiled binary (for x86_64). I tried to create a derivation which compile the program but I'm a noob...

PS: I also tried creating a service with systemctl and it works easily, if this package request will be satisfied I'll add the service too.
